src/notification/styled-components.js

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import {styled} from '../styles/index';
+import type {StyledRootPropsT} from './types';
+import {KIND} from './constants';
+
+export const StyledRoot = styled('div', ({$theme, $kind}: StyledRootPropsT) => {
+  let color = $theme.colors.notificationPrimaryText;
+  let backgroundColor = $theme.colors.notificationPrimaryBackground;
+
+  if ($kind === KIND.success) {
```

Comment:
this seems to be a `switch...case` statement

Reply:
(I'm also used to an eslint rule that doesn't allow them 😄. This is now updated to a switch.